developers.properties

```diff
@@ -0,0 +1,6 @@
+# This file is here for 'hapi', because that "product" is missing the toplevel folder
+# (it contains the 'hapi' module directly)
+hedera-base@swirldslabs.com=Hedera Base Team
+hedera-services@swirldslabs.com=Hedera Services Team
+hedera-smart-contracts@swirldslabs.com=Hedera Smart Contracts Team
+release-engineering@swirldslabs.com=Release Engineering Team
```

Comment:
we are phasing out `swirldlabs.com` and moving to `hashgraph.com`

Reply:
Would it be the same addresses for the teams, just replace the domain? Then I can do that as part of this PR.

Reply:
I think so, maybe @poulok knows, or @nathanklick